### PR TITLE
(docs) update docs/design.md

### PR DESCRIPTION
**Pull request by vladdoster**
- update spelling
- update agreement
- correct capitalization
- remove redundancy

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -29,32 +29,31 @@ with the "cloud" (12-factor) way of doing things, as well as a clear way of comm
...
 
-In side your container, you can replace any of our code with your own to make it behave exactly as you wish. You could even create one dedicated shell per cluster with logic tailored specifically for that cluster.
+Inside your container, you can replace any of our code with your own to make it behave exactly as you wish. You could even create one dedicated shell per cluster with logic tailored specifically for that cluster.
 
 ### Here are some other tips.
 
-1. Many of our `Makefiles` do an `-include Makefile.*`, which means, we'll include other `Makefiles` in that directory. To add additional functionality, simply drop-in your `Makefile.something` in that directory.
+1. Many of our `Makefiles` do an `-include Makefile.*`, which means, we'll include other `Makefiles` in that directory. To add additional functionality, drop in your `Makefile.something` in that directory.
```
Comment (Nuru) [inline docs/design.md:52]:
```suggestion
1. Many of our `Makefiles` do an `-include Makefile.*`, which means, we'll include other `Makefiles` in that directory. To add additional functionality, simply add a `Makefile.something` file in that directory.
```

**Review by Nuru — CHANGES_REQUESTED** [review]
Thank you for taking the time and effort to clean up this document. Please accept the few small changes I am requesting and we will be good to go.

**Comment by vladdoster** [comment]
I'm almost positive the `Labeler / label (pull_request)` failure isn't related to these changes.

Going to cross my i's and cross my t's by squashing these two additional commits.

Thanks for the feedback and review!

**Comment by Nuru** [comment]
> I'm almost positive the Labeler / label (pull_request) failure isn't related to these changes.

@vladdoster Correct, Labeler simply does not work on forks, because you do not have permission to add labels to this repo. CC: @osterman

**Review by Nuru — APPROVED** [review]

**Pull request merged**